src/ifcblenderexport/blenderbim/bim/ui.py

```diff
@@ -917,6 +959,56 @@ def draw(self, context):
             row.operator("bim.set_north_offset")
 
 
+class BIM_PT_presentation_layers(Panel):
+    bl_label = "IFC Presentation Layers"
+    bl_idname = "BIM_PT_presentation_layer"
+    bl_options = {"DEFAULT_CLOSED"}
+    bl_space_type = "PROPERTIES"
+    bl_region_type = "WINDOW"
+    bl_context = "scene"
+
+    def draw(self, context):
+        layout = self.layout
+        layout.use_property_split = True
+        props = context.scene.BIMProperties
+
+        layout.row().prop(props, "presentation_layers")
+        layout.row().operator("bim.add_presentation_layer")
+
+        if props.presentation_layers:
+            layout.template_list(
+                "BIM_UL_presentation_layers", "", props, "presentation_layers", props, "active_presentation_layer_index"
+            )
+            pres_layer = props.presentation_layers[props.active_presentation_layer_index]
+
+            row = layout.row(align=True)
+            row.prop(pres_layer, "name")
+            if pres_layer.name in bpy.context.scene.BIMProperties.presentation_layers:
+                pres_layer = bpy.context.scene.BIMProperties.presentation_layers[pres_layer.name]
```

Comment:
This will not work if two layers have the same name. You can trigger this bug by adding two layers and not changing their name, and then editing the description of one but not the other. A better approach is to use props.active_presentation_layer_index